website/content/docs/integrations/vault/index.mdx

```diff
@@ -53,8 +53,22 @@ Vault is configured to act as the certificate authority (CA), to ensure that the
 
 `/sign` - The Boundary controllers generate the key pair and then send it to Vault to sign.
 
+When you connect to a target, which uses Vault for dynamic SSH certificates, a new certificate is generated for every target connection. As long as the target trusts the CA, then access is granted without you having any visibility into the credentials involved.
+
+# Vault clients
```

Comment:
I'm wondering if we can get rid of this heading since there is no content below it. Alternatively, we could make this a level 2 heading (there should only be 1 level 1 at the top of the page) and add a short introductory paragraph about how you could use Vault clients for secret management and IdP, and then make the following 2 sections level 3 headings.

Reply:
Thanks Dan, yes I didn't know how best to handle this, so just left it how it was as I knew you'd have some recommendations. I am not precious about it, so whatever you think is best, based on your expert knowledge